exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -137,7 +147,8 @@ public void testFoldLeftOnEmptyList() {
                 ListOps.foldLeft(
                     Collections.<Double>emptyList(),
                     2.0,
-                    (x, y) -> x * y));
+                    (x, y) -> x * y)
```

Comment:
Technically the closing bracket at the end of this line should also be on a new line, to be completely consistent, as that closes the `foldLeft` which is spread over several lines. The same applies to the other tests using `foldLeft` and `foldRight` spread over several lines. It's not very important though, so I'm happy for this to be merged as it is if you prefer 🙂

Reply:
Ah yes, I missed it. Now all the cases should be covered.